### PR TITLE
Update 01-api-definition.md

### PR DESCRIPTION
**Pull request by tkurian123**
Being more specific on Expedia's photo requirements. Technically accepted vs Expedia Preference.

Preview available @ http://tkurian123-patch-3.partner-api-preview.test.expedia.com/apis/product-management/property-api-contact-expedia-before-adopting-/api-definition.html

### CODE REVIEW

File: apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md [hunk 1]
```diff
@@ -279,9 +279,15 @@ Contact set data are represented as a dictionary, with the following dictionary
 
 **image**
 
+While a property needs at least 1 image to complete onboarding, we recommend all partners to abide to the Expedia Standards to ensure good content:
+
+Photo Quantity: A minimum of 20 property photos.
+GuestRoom Coverage: 4 photos of each roomtype including 1 photo of the bathroom
+Photo Quality: Images below 350x350 pixels will be rejected.To ensure your photos look beautiful on desktop, tablet, and mobile screens we require at least 2,880 pixels on the longest side.
```
Comment (a-jomills) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md]: Needs a space here: "will be rejected. To ensure"

Comment (tkurian123) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md]: Done..

**Comment by a-jomills** [comment]
👍

**Comment by a-jomills** [comment]
I added the preview link to the summary.

**Comment by jgorman-exp** [comment]
Thanks, @tkurian123!

**Pull request merged**